sdk/keyvault/Azure.Security.KeyVault.Keys/tests/DelayCreateKeyInterceptor.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using Azure.Core.Testing;
+using Castle.DynamicProxy;
+using NUnit.Framework;
+
+namespace Azure.Security.KeyVault.Keys.Tests
+{
+    // TODO: Remove this implementation when https://github.com/Azure/azure-sdk-for-net/issues/8575 is fixed.
+    internal class DelayCreateKeyInterceptor : IInterceptor
+    {
+        private const string DelayEnvironmentVariableName = "AZURE_KEYVAULT_TEST_DELAY";
```

Comment:
Who would set this?

Reply:
It allows us to override it in the pipeline as a variable, so we can adjust without commit as needed.